README.md

```diff
@@ -594,6 +594,23 @@ Example oozie wf snippet:
 </action>
 ```
 
+## Other tools included
+
+### Configuration parser check
+
+`com.target.data_validator.ConfigParser` has an entrypoint that will check that the configuration file
+is parseable. It _does not_ validate variable substitutions since those have runtime implications.
+
+```shell
+spark-submit \
```

Comment:
Can this run w/o spark submit?

Reply:
Not easily. I tried it. I ran into the need to add the Spark jars to the classpath. `spark-submit` handles that. Probably just best to play in the Spark sandbox entirely.